### PR TITLE
Add io.github.RodZill4.Material-Maker

### PR DESCRIPTION
**Pull request by Calinou**
Material Maker is a tool that can be used to create textures procedurally for use in PBR rendering engines.

### CODE REVIEW

File: io.github.RodZill4.Material-Maker.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+app-id: io.github.RodZill4.Material-Maker
+base: org.godotengine.godot.BaseApp
+base-version: "3.2"
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+default-branch: stable
```
Comment (bilelmoussaoui) [inline io.github.RodZill4.Material-Maker.yaml]:
```suggestion
```

**Comment by bilelmoussaoui** [comment]
bot, build io.github.RodZill4.Material-Maker

**Comment by bilelmoussaoui** [comment]
Any idea if upstream is interested in (co-)maintaining the flatpak package?

**Comment by barthalion** [comment]
ping @Calinou, have you managed to contact the author?

**Comment by Calinou** [comment]
cc @RodZill4

**Comment by RodZill4** [comment]
Unfortunately I don't have a Linux PC to test this (only use VMs, and it's a PITA).

**Comment by barthalion** [comment]
@RodZill4 Do you mind if we grant you write access to the repository? Just in case, as it's yours application even if @Calinou will handle the maintenance.

**Comment by RodZill4** [comment]
That's perfect. Anyway I have to install Linux somewhere, testing Linux releases is way too painful with VMs.

**Pull request closed without merging**